lib/stat_tracker.rb

```diff
@@ -0,0 +1,23 @@
+require 'csv'
+class StatTracker
+  attr_reader :games_data,
+              :game_team_data,
+              :team_data
+
+  def initialize(games_data, game_team_data, team_data)
+    @games_data = games_data
+    @game_team_data = game_team_data
+    @team_data = team_data
+  end
+
+  def self.from_csv(locations)
+    games_data = CSV.open locations[:games], headers: true, header_converters: :symbol
```

Comment:
Will need to add a class or classes that parse this data into attributes that we can use for methods